src/app/features/shows/pages/movie-list/movie-list.component.ts

```diff
@@ -17,6 +18,8 @@ export class MovieListComponent implements OnInit, OnDestroy {
   searchTextUpdate = new Subject<string>();
   loading: boolean = true;
   getMoviesByApiSub: Subscription;
+  length: number;
```

Comment:
```suggestion
  totalResults: number;
```
length tối nghĩa quá e